### DISCUSSION TITLE
Is there any official site maintenances rhino docs?

### DISCUSSION THREAD

**Discussion by MaxisTekfield** [mozilla/rhino#1110] [Q&A] [answered]
I've been confused for a long time. Since the [origin site](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/Rhino/Documentation) maintenanced docs for rhino has been obsoleted and is not available, 
1. why don't we move them to a new place and keep updating them?
2. do we have plan to publish or update docs for embedders?

As [README.md](https://github.com/mozilla/rhino/blob/master/README.md) says, we can only obtain docs via [3rd-party archivings](http://web.archive.org/web/20210304081342/https://developer.mozilla.org/en-US/docs/Mozilla/Projects/Rhino/Documentation) which may be not stable. I am not sure whether it can survive for many years.
Most importantly, we are not able to update those docs now. For example, `ClassShutter` has never been mentioned in docs.

**Comment by MaxisTekfield** [comment, marked as answer]
I found #954 :)

**Reply by p-bakker** [reply]
We also have https://github.com/mozilla/rhino/projects/3

Added a note there in the TODO column to also add documentation about the ClassShutter
